lib/Hero/Content.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button as ButtonComponent } from '../Button';
+
+const Content = ({
+  classNames = {},
+  title,
+  subTitle,
+  imageUrl,
+  hasOverlay,
+  NextImage,
```

Comment:
Let's not reference `next/image` anywhere here. The idea is for it to be platform agnostic. The main goal is to be able to use `next/image`, but also any other image component in any other React framework.

Let's be consistent with our other components and call it `Image` instead.

Reply:
Got it, Working on it